netlify/netlify-plugin-nextjs
# fix: correct redirect priority and correctly handle ISR pages assets

### Summary

Corrects the ordering of rewrites, and adds one to ensure api routes aren't served by ODBs. Also ensures that pre-rendered ISR pages aren't served from the CDN, as they need to be handled by the function.

### Test plan

1. Visit the [the deploy preview api route](https://deploy-preview-826--netlify-plugin-nextjs-demo.netlify.app/api/hello?anyparam=anything) and ensure that the query is echoed back
2. Visit [a revalidate page](https://deploy-preview-826--netlify-plugin-nextjs-demo.netlify.app/getStaticProps/withRevalidate/1/) and check in the network panel that the header `x-render-mode` is `ssr` and the `date` header is now. (Don't expect the render time in the page to update)


### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

Fixes #765 


🧪 Once merged, make sure to update the version if needed and that it was published correctly.
